src/components/BorrowerProfile/LoanStory.vue

```diff
@@ -196,7 +196,7 @@ export default {
 		}`;
 
 		try {
-			const data = this.apollo.readQuery({
+			const data = this.apollo.query({
```

Comment:
`query` is asynchronous. It returns a promise for the data, not the actual data.

Reply:
Fixed, and data return format confirmed.